build.sbt

```diff
@@ -45,7 +45,6 @@ lazy val common = project
       Dependencies.Libraries.jaywayJsonpath,
       Dependencies.Libraries.iabClient,
       Dependencies.Libraries.yauaa,
-      Dependencies.Libraries.rhino,
```

Comment:
something's wrong here - where did these dependencies come from that you had to delete? When I check the Snowplow Enrich master I don't see either of them https://github.com/snowplow/enrich/blob/master/build.sbt

Reply:
Ya not sure, I too checked the master branch and didn't find them, hence deleted them ( also they were causing the issue if present). 
Need to do due-diligence to figure it out.

Reply:
I think you need to fix this before you go any further. For example the build.sbt in your repo's master branch is from last June, while the version in the public Snowplow repo is from October, and has many changes from the June version.
Ideally we want to be able to submit these changes back to the Snowplow project as a PR for their repo, so we had better make sure we are basing the changes on their most recent release.